app/src/main/kotlin/com/wire/android/ui/common/WireDialog.kt

```diff
@@ -200,21 +201,27 @@ private fun WireDialogContent(
                 }
             }
             text?.let {
-                ClickableText(
-                    text = text,
-                    style = MaterialTheme.wireTypography.body01,
-                    modifier = Modifier.padding(
-                        top = MaterialTheme.wireDimensions.dialogTextsSpacing,
-                        bottom = MaterialTheme.wireDimensions.dialogTextsSpacing,
-                    ),
-                    onClick = { offset ->
-                        text.getStringAnnotations(
-                            tag = MarkdownConstants.TAG_URL,
-                            start = offset,
-                            end = offset,
-                        ).firstOrNull()?.let { result -> uriHandler.openUri(result.item) }
+                LazyColumn(modifier = Modifier
```

Comment:
nitpick: regular `Column` with `scrollable` modifier I think would be enough, but this is also fine

Reply:
Question: does this add the sliding indicator on the side?